roadmap/2021.md

```diff
@@ -104,30 +104,30 @@ Hacktoberfest
 SDKs:
 - [X] Rust SDK: normalized + error handler
 - [X] Instant MeiliSearch: add cache
-- [ ] Java SDK: clean the double implementation of some routes
+- (_Delayed to 2022_) Java SDK: clean the double implementation of some routes
 
 Front-End:
-- [X] Finish Gatsby integration
-- [ ] (?) Write the "Gatsby + MeiliSearch" article
+- (_Released at the beginning of 2022_) Finish Gatsby integration
+- (_Delayed_) Write the "Gatsby + MeiliSearch" article
```

Comment:
@shivaylamba is on it, see the issue [in the Devrel repo](https://github.com/meilisearch/devrel/issues/340)